clients/src/main/java/org/apache/kafka/clients/admin/NewTopic.java

```diff
@@ -98,6 +98,13 @@ public NewTopic configs(Map<String, String> configs) {
         return this;
     }
 
+    /**
+     * The configuration for the new topic or null if no configs ever specified.
+     */
+    public Map<String, String> configs() {
```

Comment:
public API change? is this in the KIP?

Reply:
Yes I'm planning to update the KIP and calls for a re-vote including this change.